AdaptorsFunctionality/test/unit/src/au/gov/asd/tac/constellation/functionality/adaptors/dataaccess/plugins/importing/ImportFromRDFPluginNGTest.java

```diff
@@ -51,46 +75,80 @@ public void setUpMethod() throws Exception {
     public void tearDownMethod() throws Exception {
     }
 
-    /**
-     * Test of query method, of class ImportFromRDFPlugin.
-     */
+
     @Test
-    public void testQuery() throws Exception {
+    public void testMusicTurtleQuery() throws Exception {
         final RecordStore query = new GraphRecordStore();
-        PluginInteraction interaction = null;
-        PluginParameters parameters = new PluginParameters();
-        ImportFromRDFPlugin instance = new ImportFromRDFPlugin();
-        RecordStore expResult = new GraphRecordStore();
-        RecordStore result = instance.query(query, interaction, parameters);
-        assertEquals(expResult, result);
+        final PluginInteraction interaction = null;
+        final ImportFromRDFPlugin instance = new ImportFromRDFPlugin();
+        final PluginParameters parameters = instance.createParameters();
+        parameters.setStringValue(ImportFromRDFPlugin.INPUT_FILE_URI_PARAMETER_ID, "file://" + this.getClass().getResource("./resources/music.ttl").getFile());
+
+        final RecordStore expResult = new GraphRecordStore();
+        final RecordStore result = instance.query(query, interaction, parameters);
+//        System.out.println(result.toStringVerbose());
```

Comment:
The output from this can be used to re-construct the `RecordStore` that can be used to verify the final `RecordStore`.